pom.xml

```diff
@@ -31,40 +31,46 @@
         <dependency>
             <groupId>net.shibboleth.idp</groupId>
             <artifactId>idp-authn-api</artifactId>
-            <version>4.0.1</version>
+            <version>5.1.3</version>
         </dependency>
 
         <dependency>
             <groupId>net.shibboleth.idp</groupId>
             <artifactId>idp-session-api</artifactId>
-            <version>4.0.1</version>
+            <version>5.1.3</version>
         </dependency>
 
         <dependency>
             <groupId>net.shibboleth.utilities</groupId>
             <artifactId>java-support</artifactId>
-            <version>8.1.0</version>
+            <version>8.4.2</version>
         </dependency>
 
         <dependency>
-            <groupId>javax.json</groupId>
-            <artifactId>javax.json-api</artifactId>
-            <version>1.0</version>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
+            <version>2.1.3</version>
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+
         <dependency>
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
-            <version>1.2</version>
+            <version>1.3.3</version>
         </dependency>
 
        <dependency>
           <groupId>edu.washington.shibboleth</groupId>
           <artifactId>tools</artifactId>
-          <version>1.2.1</version>
+          <version>1.3.0</version>
           <scope>system</scope>
-          <systemPath>/data/local/src/uw-http-client/target/uw-http-client-1.2.1.jar</systemPath>
+          <systemPath>/data/local/src/uw-http-client/target/uw-http-client-1.3.0.jar</systemPath>
```

Comment:
just noting that if we had CI/CD setup with uw-http-client (repo) that kept our Google Artifact Registry (has maven) then we could source that instead of a local file system.